storage/src/context.rs

```diff
@@ -126,7 +126,7 @@ impl ContextApi for TezedgeContext {
 
     fn get_key_from_history(&self, context_hash: &ContextHash, key: &ContextKey) -> Result<Option<ContextValue>, ContextError> {
         let context_hash_arr: EntryHash = context_hash.as_slice().try_into()?;
-        let merkle = self.merkle.read().expect("lock poisoning");
+        let mut merkle = self.merkle.write().expect("lock poisoning");
```

Comment:
@michal-stanek 
why do we need `write lock` for just reading from history fixed data?

Reply:
I had to change all read locks to write because we're updating statistics on each operation.